data/json/effect_on_condition.json

```diff
@@ -169,6 +169,26 @@
     },
     "effect": [ { "u_message": "NIGHT_MESSAGES_3", "snippet": true } ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "eoc_robofac_surveillance",
+    "condition": { "or": [ { "u_has_trait": "PROF_HUB01_ANCILLIARY" }, { "u_has_effect": "robofac_surveillance" } ] },
+    "deactivate_condition": { "not": { "or": [ { "u_has_trait": "PROF_HUB01_ANCILLIARY" }, { "u_has_effect": "robofac_surveillance" } ] } },
+    "effect": [
+      {
+        "u_set_spawn_monster": "mon_robofac_camspy",
+        "real_count": 1,
+        "min_radius": 15,
+        "max_radius": 40,
+        "outdoor_only": true
```

Comment:
Should this be limited (for the initial spawn) to daytime? If not, the vision_night may need increasing further - or, possibly better, give the spybot HEARS and GOODHEARING, which means it won't necessarily be great at _finding_ the PC, but may at least be spotted heading towards something of interest.